pep-0526.txt

```diff
@@ -287,11 +308,9 @@ Changes to standard library and documentation
 Runtime effects of type annotations
 ===================================
 
-As stated under "Variable Annotations", annotating a local variable will cause
+Annotating a local variable will cause
 the interpreter to treat it as a local, even if it was never assigned to.
-
-If a variable annotation is for a local variable, the annotation will not be
-evaluated::
+Annotation for a local variable will not be evaluated::
```

Comment:
This sounds weird to me...it should be either "Annotations for locals variables will not be evaluated" or "An annotation for a local variable will not be evaluated".